### PR TITLE
update spirv_ vulkan_ gtk3 gtk4 glslang

### PR DESCRIPTION
**Pull request by satmandu**
Builds properly:
- [x] `x86_64`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=vulkan_spirv_gtk_updates crew update
```

### CODE REVIEW

File: packages/spirv_tools.rb [hunk 1]
```diff
@@ -5,54 +5,43 @@
 class Spirv_tools < Package
```
Comment (uberhacker) [inline packages/spirv_tools.rb:5]: Use Cmake buildsystem?

File: packages/vulkan_icd_loader.rb [hunk 2]
```diff
@@ -3,21 +3,21 @@
 class Vulkan_icd_loader < Package
```
Comment (uberhacker) [inline packages/vulkan_icd_loader.rb:3]: Use CMake buildsystem?

File: packages/vulkan_tools.rb [hunk 3]
```diff
@@ -6,21 +6,21 @@
 class Vulkan_tools < Package
```
Comment (uberhacker) [inline packages/vulkan_tools.rb:6]: Use CMake buildsystem?

**Review by uberhacker — DISMISSED** [review]
Use CMake buildsystem?

File: packages/vulkan_tools.rb (hunk 3 above)
Comment (satmandu) [reply packages/vulkan_tools.rb:6]: This doesn't really work. It's too complicated, as it is a dual build...

**Review by uberhacker — APPROVED** [review]

**Pull request merged**